packages/electron/src/preload.ts

```diff
@@ -91,6 +91,10 @@ contextBridge.exposeInMainWorld("electron", {
   verifyUser: (profile: string) => invokeBackend("verifyProfile", profile),
   getVersion: () => invokeBackend("getVersion"),
   telemetry: (message: any) => invokeBackend("telemetry", message),
+  getAssessmentLog: () => {
+    const dateString = new Date().toISOString().slice(0,10).replace(/-/g,"");
```

Comment:
Are we generating this same filename somewhere else? If yes, can we reuse from that location?

Reply:
The filename is coming from the log configuration, so I don't think it's usable in this context.

https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs#L39